src/bit_set.rs

```diff
@@ -15,26 +15,21 @@ use std::slice;
 #[derive(Default,Debug)]
 pub struct Bitset {
     head: usize,
-    tail: Option<Box<Vec<usize>>>,
+    tail: Option<Vec<usize>>,
```

Comment:
Note that the `Box<>` here was redundant, since `Vec<T>` is already essentially `Box<[T]>` (and in particular it is `Sized`).